spring-cloud-open-service-broker-core/src/test/java/org/springframework/cloud/servicebroker/controller/ServiceInstanceBindingControllerResponseCodeTest.java

```diff
@@ -27,6 +27,7 @@
 import org.junit.experimental.theories.Theories;
 import org.junit.experimental.theories.Theory;
 import org.junit.runner.RunWith;
+import org.springframework.cloud.servicebroker.exception.ServiceInstanceDoesNotExistException;
```

Comment:
It would be great to follow to the general style and move this line a little bit lower - to spring's imports.